packit_service/worker/reporting/reporters/pagure.py

```diff
@@ -47,8 +47,16 @@ def set_status(
             url = CONTACTS_URL
 
         if self.pull_request_object:
+            # generate a custom uid from the check_name,
+            # so that we can update flags we set previously,
+            # instead of creating new ones (Pagure specific behaviour)
+            uid = hashlib.md5(check_name.encode()).hexdigest()
```

Comment:
I would recommend staying away from MD5 (or SHA1), it could potentially disappear in the future.

```suggestion
            uid = hashlib.sha256(check_name.encode()).hexdigest()
```